src/Native/Runtime/CMakeLists.txt

```diff
@@ -216,8 +221,8 @@ convert_to_absolute_path(PORTABLE_RUNTIME_SOURCES ${PORTABLE_RUNTIME_SOURCES})
 
 convert_to_absolute_path(RUNTIME_SOURCES_ARCH_ASM ${RUNTIME_SOURCES_ARCH_ASM})
 
-if(NOT CLR_CMAKE_PLATFORM_WASM)
-  add_subdirectory(Full)
-endif(NOT CLR_CMAKE_PLATFORM_WASM)
+if(NOT CLR_CMAKE_PLATFORM_ARCH_WASM)
+    add_subdirectory(Full)
```

Comment:
Nit: This file seems to be using two space indentation

Reply:
Fixed